src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -81,8 +81,13 @@ private void UpdateInstalledVersions()
         {
             var hash = new HashSet<NuGetVersion>();
 
+            IsRequestedVisible = false;
```

Comment:
I'd use `_isRequestedVisible ` here so that we don't fire PropertyChanged unnecessarily.

Reply:
Better yet... You could use a local like `bool anyRequested = false`, then set it once down here https://github.com/NuGet/NuGet.Client/pull/3555/commits/bfd4f346c9c377deeab4ac8b2098aa6aea538855#diff-17f11a93b3135e66bd31d0d723de815aR122 (eg, `IsRequestedVisible = anyRequested`)